deploy/charts/rook-ceph/values.yaml

```
@@ -66,6 +66,8 @@ csi:
   enableCSIHostNetwork: true
   # set to false to disable deployment of snapshotter container in CephFS provisioner pod.
   enableCephfsSnapshotter: true
+  # set to false to disable deployment of snapshotter container in NFS provisioner pod.
+  enableNFSSnapshotter: true
```

Since `ROOK_CSI_ENABLE_NFS` is false by default, seems like the nfs snapshotter should also default to false. And the same default should be changed in other places.
```suggestion
  enableNFSSnapshotter: false
```

---

> Since `ROOK_CSI_ENABLE_NFS` is false by default, seems like the nfs snapshotter should also default to false. And the same default should be changed in other places.

This will only come into affect when nfs csi pods are enabled, so I think we can keep the default as true similar to other plugins.